src/amber/exceptions/exceptions.cr

```diff
@@ -6,6 +6,12 @@ module Amber
       end
     end
 
+    class InvalidRouteAction < Exception
```

Comment:
I think is good where it is now since this will error while defining the routes and not when processing a request.